app/src/androidTest/java/ch/hikemate/app/navigation/SideBarNavigationTest.kt

```diff
@@ -353,7 +364,8 @@ class SideBarNavigationTest {
   @Test
   fun navigationItemDrawer_displaysLabel() {
     composeTestRule.setContent {
-      SideBarNavigation(onTabSelect = {}, tabList = LIST_TOP_LEVEL_DESTINATIONS, selectedItem = "")
+      SideBarNavigation(
+          onTabSelect = {}, tabList = LIST_TOP_LEVEL_DESTINATIONS, selectedItem = "") {}
```

Comment:
Question: Do I observe correctly, that you didn't actually change anything here?

Reply:
I just added the {} for the content